src/common/__tests__/__snapshots__/Button-test.js.snap

```diff
@@ -2,19 +2,17 @@ exports[`test renders correctly 1`] = `
 <View
```

Comment:
Will remove it, and we should investigate where snapshot testing makes sense. It seems it does wth redux and other use-cases and maybe for visual components once stuff settles down.